DartConfiguration.tcl

```diff
@@ -0,0 +1,106 @@
+# This file is configured by CMake automatically as DartConfiguration.tcl
+# If you choose not to use CMake, this file may be hand configured, by
+# filling in the required variables.
```

Comment:
Q: What is this file used for? is it really needed or even used?
"tcl" is for a scripting language called "tool control language".
it is an olde-timey scripting language.
i think most people use python these days.